app.py

```diff
@@ -32,20 +34,13 @@ def send_reminder():
     send_messages(indicator, message)
     return render_template("message-success.html")
 
-
 @app.route("/send-lifemap", methods=["POST"])
 def send_lifemap():
-    phone_number = request.form["from"]
-    print(phone_number)
-
-    # Save the number to our database
-    db = connect_mongo()
-    numbers = db["numbers"]
-    numbers.insert({"number":phone_number})
-
-    twilio_send_template(phone_number)
-    return "lifemap sent!"
-
+    phone_number = request.form['From']
+    save_number(phone_number)
+    # pdfnetor(phone_number)
+    lifemap = send_pdf(phone_number)
+    return jsonify("pdf successfully sent")
```

Comment:
If privacy isn't a concern, might be helpful to include the phone number in the return statement (e.g. "pdf successfully sent to " + phone_number)

Reply:
Would be a good idea to log this somewhere for future debugging/data purposes